diagnostic.md

```diff
@@ -3,36 +3,44 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Low bandwith environments with single page applications.
+You can control state with API only development.
```

Comment:
I'm not sure what you mean by these first two.

The main things are:
Front-end routing
Displaying content dynamically using two-way data-binding
No more AJAX - Ember makes API requests for you.

0/1